src/api/app/controllers/webui/users_controller.rb

```diff
@@ -179,6 +183,12 @@ def set_all_filters_if_none_is_set(filters, filter_keys)
     end
   end
 
+  def authorize_policy_initialize
+    policy(User)
+    # TODO: Remove when we'll do the authorization checks with Pundit
+    skip_authorization
```

Comment:
This may look a bit odd so let me explain the reasons: Currently in [UserPolicy](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/policies/user_policy.rb#L2-L8) we have some authorization rules used for the `api` which differs for the ones that we need for the `webui`. Adding the authorization rules for the `webui` in this PR (whilst still keeping the other rules for the `api`) would've been a bit complicated and out of scope.

The `skip_authorization` is needed because we don't call `authorize` yet (the logged in check is in `initialize`)

Reply:
Could we instead have a user policy for webui and another for the api? I would find this easier to understand when looking at this again in a few weeks/months. We already do this for controllers. The api controllers are directly under `src/api/app/controllers/` while the webui controllers are under `src/api/app/controllers/webui/`. So the policies for the api would be under `src/api/app/policies/` and the ones for webui would be under `src/api/app/policies/webui/`.

What do you think?

Reply:
Agree! We'll have to differentiate the 2 policies logic at some point and using different classes looks pretty straightforward 👍  

For the sake of this refactor is fine if we don't port the authorization logic from the controllers into the `webui` policy yet (I'd just port the `require_login` logic for now)?

Reply:
It's fine to just port the `require_login` logic for now.

Reply:
@dmarcoux I've created a separate policy now, could you please check?